### PR TITLE
docs(recipes): update rsync-deploy

### PR DESCRIPTION
**Pull request by antleblanc**
Update recipe for rsync-deploy.

see: #742

Signed-off-by: Antoine Leblanc <ant.leblanc@gmail.com>

### CODE REVIEW

File: docs/recipes/rsync-deploy.md [hunk 1]
```diff
@@ -13,46 +13,47 @@ $ npm install --save-dev gulp-rsync
...
 
+Add a file to project root `rsync.json`.
 
-```
+```json
 {
-    "hostname": "hostname or ip",
-    "username": "username",
-    "destination": "folder"
+  "hostname": "hostname or ip",
```
Comment (antleblanc) [inline docs/recipes/rsync-deploy.md:21]: According to the [EditorConfig](https://github.com/yeoman/generator-webapp/blob/master/app/templates/editorconfig#L12) file, I switched to 2 spaces for JSON files.

File: docs/recipes/rsync-deploy.md [hunk 2]
```diff
@@ -13,46 +13,47 @@ $ npm install --save-dev gulp-rsync
 ```
 
 ### 2. Create a `rsync config` file
-Add a file to project root `rsync.json`.
 
+Add a file to project root `rsync.json`.
 
-```
+```json
```
Comment (antleblanc) [inline docs/recipes/rsync-deploy.md:19]: Turn on syntax color highlighting.

File: docs/recipes/rsync-deploy.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Rsync Deploying
 
-This is an easy way publish your site with rsynch to serve the generated static files.
+This is an easy way publish your site with [rsync](https://rsync.samba.org/) to serve the generated static files.
```
Comment (antleblanc) [inline docs/recipes/rsync-deploy.md:3]: Provides a link to the official website of the utility.

**Review by silvenon — APPROVED** [review]
Thanks! ❤️

**Pull request merged**